packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-cards.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, h, JSX, Element } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+
+/**
+ * @slot - Slot for selector cards
+ */
+
+@Component({
+  styleUrl: 'gux-selector-cards.scss',
+  tag: 'gux-selector-cards-beta',
+  shadow: true
+})
+export class GuxSelectorCards {
```

Comment:
I'm trying to think if we could combine gux-selector-cards and gux-selector-card into just one component instead of having a separate component just for gux-selector-cards. Thoughts?